pkg/sourcereader/embedded.go

```diff
@@ -53,8 +53,8 @@ func copyFileOut(bfs BaseFS, src string, dst string) error {
 	return nil
 }
 
-// copyDirFromModules copies an FS directory to a local path
-func copyDirFromModules(bfs BaseFS, source string, dest string) error {
+// copyDir copies an FS directory to a local path
+func copyDir(bfs BaseFS, source string, dest string) error {
```

Comment:
Just a question about this function, how does isDIr handle links?  Is it possible that a link to a parent folder would cause the recursive function to infinite loop and OOM?

Reply:
Good point, created a bug to look at it.